custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -128,6 +134,49 @@ def allowed_to_forward(self, episode_case):
         )
 
 
+class NikshayFollowupRepeater(CaseRepeater):
+    followup_for_tests = ['end_of_ip', 'end_of_cp']
+
+    class Meta(object):
+        app_label = 'repeaters'
+
+    include_app_id_param = False
+    friendly_name = _("Forward eNikshay Patient's Follow Ups to Nikshay")
```

Comment:
I have added case types to the index itself now [here](https://github.com/dimagi/commcare-hq/pull/15430/files#diff-a358d71a8a9e9a9295ee52c447c5f9b2R4). Should we still set it in the friendly name?